validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    min_radius = 0 #@param
</pre><p>should be 5 right?</p><p>is this because dr. martinelli complained about the "105", "205", etc?</p><p>however might zero cause random values within the 5km green zone?</p><p> </p><p>...oh the min radius of 100 here should be ok.  This is the "minimum maximum" radius.</p><p>or in other words, we change the max radius to be from 100 to 3000.  we only care about max radius being 3000, so I dont think this needs to change at all.  just leave it at 100 and verify that "min_fraud_radius" is 5 (note the spelling, this is not "min_radius")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Reverted